Application/Features/WordFeatures/Queries/WordIsAlreadyExistsQuery.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Application.Features.WordFeatures.Queries.QueryExtensions;
+using Application.Interfaces;
+using Domain.Entites;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Application.Features.WordFeatures.Queries.QueriesHandlersBaseClasses;
+using MongoDB.Driver;
+
+namespace Application.Features.WordFeatures.Queries
+{
+    public class WordIsAlreadyExistsQuery : IRequest<bool>
+    {
+        public Word Word { get; set; }
+        public WordIsAlreadyExistsQuery(Word word)
+        {
+            Word = word;
+        }
+        public class WordIsAlreadyExistQueryHandler : QueriesHandlerBaseClass, IRequestHandler<WordIsAlreadyExistsQuery, bool>
+        {
+            public WordIsAlreadyExistQueryHandler(IWordsMongoDb context) : base(context)
+            {
+            }
+            public async Task<bool> Handle(WordIsAlreadyExistsQuery query, CancellationToken cancellationToken)
+            {
+                var exist = _context.Words
+                .AsQueryable()
+                .LaterThan(DateToday)
+                .ByEmail(query.Word.Email)
+                .Any(/*cancellationToken: cancellationToken*/);
```

Comment:
Check if it can be done asynchronously

Reply:
Unfortunately, using of AnyAsync + await causes an error:
![image](https://user-images.githubusercontent.com/87187545/156599713-0e232d85-dcce-489e-b408-00ec1095a6d1.png)